projects/1.20/assets/quark/quark/lang/zh_cn.json

```
@@ -0,0 +1,3364 @@
+	"advancements.quark.pat_potato.title": "土豆，土豆",
+	"advancements.quark.pat_potato.description": "驯服一只小土豆",
+	"advancements.quark.watcher_center.title": "谁在看着我？",
+	"advancements.quark.watcher_center.description": "看一眼末影侦测器",
+	"advancements.quark.crawl_in_hollow_log.title": "在树皮中爬行",
+	"advancements.quark.crawl_in_hollow_log.description": "在空心原木中爬行",
+	"advancements.quark.get_forgotten_hat.title": "博物馆的藏品",
+	"advancements.quark.get_forgotten_hat.description": "在地下杀死一只遗忘者并且获得他的帽子",
+	"advancements.quark.get_diamond_heart.title": "心的形状",
+	"advancements.quark.get_diamond_heart.description": "在地下潜行靠近石精灵并击碎它的外壳以获得钻石之心",
+	"advancements.quark.make_stoneling.title": "年轻的逐梦人",
+	"advancements.quark.make_stoneling.description": "通过对任意石头使用石精灵的钻石之心来给它一具崭新的躯体",
+	"advancements.quark.mine_toretoise.title": "矿石龟！",
+	"advancements.quark.mine_toretoise.description": "用镐子挖取野生矿石陆龟背上的矿物",
+	"advancements.quark.mine_fed_toretoise.title": "我们发财了！",
+	"advancements.quark.mine_fed_toretoise.description": "给矿石陆龟喂食发光浆果之后再次收获其背上的矿物",
+	"advancements.quark.throw_pickarang.title": "不知怎么的就回来了",
+	"advancements.quark.throw_pickarang.description": "扔出一个皮卡镖",
+	"advancements.quark.retreive_flamerang.title": "翱翔吧，火焰鸟",
+	"advancements.quark.retreive_flamerang.description": "扔出一个界炎镖并拿回一个燃烧着的物品",
+	"advancements.quark.overlevel_enchant.title": "超越极限",
+	"advancements.quark.overlevel_enchant.description": "使用古卷来超越附魔的最高等级",
+	"advancements.quark.instamine_deepslate.title": "超越瞬间的极限",
+	"advancements.quark.instamine_deepslate.description": "同时使用效率VI和急迫II来瞬间破坏深板岩",
+	"advancements.quark.redirect_beacon.title": "看见了光",
+	"advancements.quark.redirect_beacon.description": "使用刚玉簇重定向信标的光束",
+	"advancements.quark.get_bottled_cloud.title": "云存储",
+	"advancements.quark.get_bottled_cloud.description": "把云朵捕捉到瓶子中",
+	"advancements.quark.apply_color_rune.title": "五光十色",
+	"advancements.quark.apply_color_rune.description": "使用锻造台将彩色符文蚀刻应用到物品上",
+	"advancements.quark.wear_full_rainbow.title": "彩虹的味道",
+	"advancements.quark.wear_full_rainbow.description": "装备一身带有彩虹符文蚀刻效果的盔甲",
+	"advancements.quark.throw_parrot_egg.title": "只是鹦鹉",
+	"advancements.quark.throw_parrot_egg.description": "给鹦鹉喂食甜菜种子，并投掷它生下的蛋",
+	"advancements.quark.pathfinder_map_center.title": "通往明日的道路",
+	"advancements.quark.pathfinder_map_center.description": "到达寻路地图中X标号的位置",
+	"advancements.quark.get_own_head.title": "出人头地",
+	"advancements.quark.get_own_head.description": "通过利用女巫杀死被命名的雪傀儡的方式获取你自己的头颅",
+	"advancements.quark.ancient_apple_overlevel.title": "有点水果味",
+	"advancements.quark.ancient_apple_overlevel.description": "在自身经验等级超过100级的时候食用附魔水果",
+	"advancements.quark.get_all_blossom_saplings.title": "花开正茂",
+	"advancements.quark.get_all_blossom_saplings.description": "获取所有种类的喇叭树苗",
+	"advancements.quark.get_all_corundum.title": "刚玉难题",
+	"advancements.quark.get_all_corundum.description": "获取所有颜色的刚玉簇",
+	"advancements.quark.use_viaduct.title": "来追我啊",
+	"advancements.quark.use_viaduct.description": "将末影珍珠丢至一行幻境石水晶的一端来穿过它们",
```

原文【Toss an Ender Pearl to the tip of a line of Myalite Crystals to travel through them】
```suggestion
	"advancements.quark.use_viaduct.description": "将末影珍珠投掷到一行幻境石水晶的一端以快速到达另一端",
```

---

travel through 是途经的意思，就是通过末影珍珠的目的地被篡改为另一端，中间插进去了（也就是“途经”）这一系列幻境石水晶